### DISCUSSION TITLE
Can a site level column metadata be made available as a 'Slot Field' option?

### DISCUSSION THREAD

**Discussion by RogerCP66** [microsoft-search/pnp-modern-search#3664] [Q&A] [answered]
Hello,

I have a site level column that I have created and made as a managed/crawled property. I would like for that information to be one of the slots in the card layout, but it doesn't seem to add to the 'slot field' list selections (screenshot below).

The column data is available in the pnp search results it's just not coming up as a slot option.

Is there a way to change the list and make the option available?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/145495774/cda3f024-52ee-49ed-8ae3-03d5be204727)

**Comment by kasperbolarsen** [comment, marked as answer]
And I have to ask: you have added the managed property to "Selected Properties "?

**Reply by RogerCP66** [reply]
Thanks again @kasperbolarsen. That worked I was able to find the options in Selected Properties.
